src/dictionary/api.ts

```diff
@@ -61,7 +61,7 @@ export const get = async (req: Request, res: Response) => {
   return res.status(200).send(schema);
 };
 
-export const getClinicalEntities = async (req: Request, res: Response) => {
+export const getClinicalEntitiesTemplate = async (req: Request, res: Response) => {
```

Comment:
this is the `/list` right ? it can be used for anything not just templates

Reply:
oh crap, your right I don't know why I renamed this... changing back